java/arcs/core/data/testdata/example.pb.txt

```diff
@@ -0,0 +1,79 @@
+# A proto encoding of the following recipe: 
```

Comment:
Can you rename this file `example.textproto`?

Reply:
`.pb.txt` is the typical convention for the text representation of protos. So, this is probably ok, unless there are other reasons to name it `.textproto`

Reply:
I stand corrected, it seems that `.textproto` is the recommended extension. `.pb.txt` is not recognized by tools, but `.textproto` is. +1 to changing the extension.

Reply:
Done! Thanks folks!